packages/synapse-react-client/src/lib/utils/theme/DefaultTheme.ts

```diff
@@ -126,16 +136,39 @@ const defaultMuiThemeOptions: ThemeOptions = {
             )} 0 0 0 0.1rem`,
             borderColor: theme.palette.primary.main,
           },
+          '& fieldset': {
+            border: 'none',
+          },
         }),
         input: ({ theme }) => ({
           padding: '14px 12px',
         }),
+        multiline: {
+          padding: '0px',
+        },
       },
     },
     MuiInputLabel: {
       defaultProps: {
         shrink: true,
       },
+      styleOverrides: {
+        root: ({ theme }) => ({
+          fontWeight: 700,
+          mb: '4px',
```

Comment:
Styling from SRC TextField

Reply:
Thanks for getting us closer to a unified solution!